squire_lib/src/ffi_tournament.rs

```diff
@@ -58,6 +62,39 @@ impl TournamentId {
         ptr
     }
 
+    /// Returns a raw pointer to rounds
+    /// This is an array that is terminated by the NULL UUID
+    /// This is heap allocted, please free it
+    /// Returns NULL on error
+    #[no_mangle]
+    pub extern "C" fn tid_rounds(self: Self) -> *const RoundId {
+        unsafe {
+            let tourn: Tournament;
+            match FFI_TOURNAMENT_REGISTRY.get().unwrap().get(&self) {
+                Some(t) => tourn = t.value().clone(),
```

Comment:
Remove this clone. There is no need to clone the tournament.

Reply:
Fixed in 5fa7947e47d9a8eaa66478292328da63b029a448